Core/ClientSMLSWIG/Python/SConscript

```diff
@@ -17,14 +17,20 @@ python_sml_alias = clone['SML_PYTHON_ALIAS']
 
 inc_path = sysconfig.get_path('include')
 if os.name == 'nt':
-    lib_path = os.path.join(sysconfig.get_config_vars('BINDIR')[0], 'libs')
+    lib_path = [
+        os.path.join(sysconfig.get_config_vars('BINDIR')[0], 'libs'),
+        os.path.join(sysconfig.get_config_vars('LIBDEST')[0], '..\\libs'),
```

Comment:
This one is correct for the windows github runners, I'm putting them alongside the existing one, `Append` tries to combine lists as well as appending strings.